src/MicrosoftTeams.ts

```diff
@@ -745,12 +770,20 @@ namespace microsoftTeams {
          * Notifies the frame that initiated this authentication request that the request was successful.
          * This function is usable only on the authentication window.
          * This call causes the authentication window to be closed.
-         * @param result Specifies a result for the authentication. If specified, the frame that initiated the authentication pop-up receives this value in its callback.
+         * @param authenticationResultParams Specifies a result for the authentication. It contains the optional values of state(URL to redirect back after authentication in Outlook Desktop), reason(in case of failure), result(in case of success).
          */
-        export function notifySuccess(result?: string): void {
+        export function notifySuccess(authenticationResultParams: AuthenticationResultParameters): void {
+            if (authenticationResultParams.state) {
```

Comment:
I don't see a unit test for this code path (i.e. the one where auth happens via redirect). Consider adding one.

Reply:
Added Unit tests